src/components/side-panel/svg-url-clipboard.tsx

```diff
@@ -0,0 +1,100 @@
+import AlertStripe from "nav-frontend-alertstriper";
+import PanelBase from "nav-frontend-paneler";
+import * as React from "react";
+import * as CopyToClipboard from "react-copy-to-clipboard";
+import * as Redux from "react-redux";
+import Language from "../../language/norwegian";
+import { getChosenExtensions } from "../../redux/selectors";
+import { IconExpanded, IconStyle, Store } from "../../redux/store-interfaces";
+import { iconCDNRelative } from "../../utils/api-link-creator";
+import "./svg-url-clipboard.less";
+
+interface PropTypes {
+  icon: IconExpanded;
+  chosenExtensions: string[];
+  iconStyle: IconStyle;
+  iconColor: string;
+}
+
+interface StateTypes {
+  copied: boolean;
+}
+
+class SvgUrlClipboard extends React.Component<PropTypes, StateTypes> {
+  constructor(props: PropTypes) {
+    super(props);
+    this.state = {
+      copied: false
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  public handleClick() {
+    this.setState({ copied: true });
+  }
+
+  public shouldComponentUpdate(nextProps: PropTypes, nextState: StateTypes) {
+    return (
+      this.props.icon.id !== nextProps.icon.id ||
+      this.state.copied !== nextState.copied ||
+      this.props.iconColor !== nextProps.iconColor
+    );
+  }
+
+  public componentWillReceiveProps(props: PropTypes) {
```

Comment:
Trenger vi denne?

Reply:
Når man klikker på knappen som kopierer til clipboard, erstattes den med en suksess-alertstripe. componentWillReceiveProps er nødvendig for å få tilbake kopier-knappen når man klikker på nytt ikon/farge/type